doc/mgr/rook.rst

```diff
@@ -1,4 +1,6 @@
 
+.. _mgr-rook:
+
 =============================
 Rook orchestrator integration
 =============================
```

Comment:
Update this as well. Because this shows up as
```
The deployment of the nfs-ganesha daemons can also be managed automatically if either the Cephadm or Rook orchestrator integration orchestrators are enabled. 
```